index.html

```diff
@@ -222,46 +222,46 @@ <h3>Specific Verifiable Credentials</h3>
 that should work for most use cases.
         </p>
       </section>
+    </section>
+    <section>
+      <h2>Disputes</h2>
 
-      <section>
-        <h3>Disputes</h3>
-
-        <p>
-  There are at least two different cases to consider where an <a>entity</a>
-  wants to dispute a <a>credential</a> issued by an <a>issuer</a>:
-        </p>
+      <p>
+There are at least two different cases to consider where an <a>entity</a>
+wants to dispute a <a>credential</a> issued by an <a>issuer</a>:
+      </p>
 
-        <ul>
-          <li>
-  A <a>subject</a> disputes a claim made by the <a>issuer</a>. For example, the
-  <code>address</code> <a>property</a> is incorrect or out of date.
-          </li>
-          <li>
-  An <a>entity</a> disputes a potentially false claim made by the <a>issuer</a>
-  about a different <a>subject</a>. For example, an imposter claims the social
-  security number for an <a>entity</a>.
-          </li>
-        </ul>
+      <ul>
+        <li>
+A <a>subject</a> disputes a claim made by the <a>issuer</a>. For example, the
+<code>address</code> <a>property</a> is incorrect or out of date.
+        </li>
+        <li>
+An <a>entity</a> disputes a potentially false claim made by the <a>issuer</a>
+about a different <a>subject</a>. For example, an imposter claims the social
```

Comment:
`claims` -> `has claimed`

Reply:
fixed in [4f030d5](https://github.com/w3c/vc-imp-guide/pull/30/commits/4f030d5ed38ae184a0e97e98b1224dac251fe653)